addons/l10n_no/data/account_tax_data.xml

```diff
@@ -992,30 +1059,31 @@
             <field name="amount_type">percent</field>
             <field name="type_tax_use">purchase</field>
             <field name="tax_group_id" ref="tax_group_25"/>
+            <field name="active" eval="False"/>
             <field name="invoice_repartition_line_ids" eval="[(5,0,0),
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'base',
-                    'plus_report_line_ids': [ref('account_tax_report_line_post_13')],
+                    'plus_report_line_ids': [ref('account_tax_report_line_code_92')],
                 }),
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'tax',
-                    'account_id': ref('chart2702'),
-                    'plus_report_line_ids': [ref('account_tax_report_line_post_13_tax')],
+                    'account_id': ref('chart2711'),
+                    'plus_report_line_ids': [ref('account_tax_report_line_code_92_tax')],
                 }),
             ]"/>
             <field name="refund_repartition_line_ids" eval="[(5,0,0),
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'base',
-                    'minus_report_line_ids': [ref('account_tax_report_line_post_13')],
+                    'minus_report_line_ids': [ref('account_tax_report_line_code_92')],
                 }),
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'tax',
-                    'account_id': ref('chart2702'),
-                    'minus_report_line_ids': [ref('account_tax_report_line_post_13_tax')],
+                    'account_id': ref('chart2711'),
+                    'minus_report_line_ids': [ref('account_tax_report_line_code_92_tax')],
                 }),
             ]"/>
         </record>
```

Comment:
That would be nice, however I'm not entirely sure how this would affect the resulting tax report lines (you're probably right, I just need to get my head around that)